R/fct_appdata.R

```diff
@@ -267,9 +267,15 @@ get_appdata <-  function(
       "item_group consists of multipe elements which is not allowed: ", 
       item_group_x
     )
+    form_type_x <- unique(with(var_levels, form_type[item_group == item_group_x]))
```

Comment:
Maybe it is good to wrap form_type_x also in `simplify_string`. That way the user is protected against some hard to detect changes in the names (for example, accidentally adding a leading or trailing white space in the table's name in the Excel file)